### PR TITLE
Update README to help with manual building

### PR DESCRIPTION
**Pull request by pancelor**
This should help anyone who ran into the problems I ran into in issue #1. 

It could also help to publish a release; https://github.com/ManuelBlanc/NecroLua/releases has no published releases (but I know you sent me a prebuilt zip for 0.3.0, so maybe you forgot to publish it?)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@
 *.ico
 *.mdmp
 *.zip
+include/*
+lib/*
```
Comment (ManuelBlanc) [inline .gitignore]: The trailing asterisks are superfluous. Can you remove them?

File: README.md [hunk 2]
```diff
@@ -29,14 +30,69 @@ Check the [wiki documentation][wiki] for more information on the modding API.
...
+
+2. After installing the above packages, go to your NecroLua directory and create an `include` subdirectory and a `lib` subdirectory, and copy in the required headers and library files; the following files should be present:
+
+```bash
+$ ls lib/ include/
+include/:
+detours.h  lauxlib.h  lua.h  luaconf.h  luajit.h  lualib.h
+
+lib/:
+detours.lib  lua51.dll  lua51.lib
```
Comment (ManuelBlanc) [inline README.md]: lua51.dll is not needed here, only with the NecroDancer.exe as you mention later.

**Review by ManuelBlanc — CHANGES_REQUESTED** [review]
Could you take a look at the requested changes before this is merged?

Thanks for the PR. You've clearly gone through quite some effort to get everything working.
I'm super stoked that you've found this project interesting ;)

File: README.md [hunk 3]
```diff
@@ -29,14 +30,69 @@ Check the [wiki documentation][wiki] for more information on the modding API.
...
+
+necrolua.hook("c_Player::p_GetElectricStrength", function(func, self)
+  return 1 - func(self)
+end)
+```
+
+This mod will make invert your character's electricity; e.g. you will do arcing electric attacks when you're _not_ on the zone 5 wire.
+
 ## Authors
 + [ManuelBlanc](https://github.com/ManuelBlanc)
```
Comment (ManuelBlanc) [inline README.md:97]: It'd be nice if you added yourself here 😉 
(Maybe rename the section to a more general _Contributors_?)

Comment (pancelor) [reply README.md:97]: :D

**Comment by pancelor** [comment]
ah, I just noticed this. thanks for the feedback!

**Pull request merged**